docs/tiling.md

```diff
@@ -0,0 +1,337 @@
+
+- **`scheme`:** The tiling scheme of the tileset.
+- **`grids`:** The URL pattern for accessing grid data.
+- **`data`:** Optional. The URL pattern for accessing data. Used for GeoJSON originally, which in this specification is replaced by MicroJSON and used in the `tiles` field.
+- **`template`:** Optional. Contains a mustache template to be used to format data from grids for interaction.
+
+## Pydantic Model for TileJSON for MicroJSON
+### TileJSON
+::: microjson.tile.TileJSON
+### TileModel
+::: microjson.tile.TileModel
+### TileLayer
+::: microjson.tile.TileLayer
+
+## TileJSON for MicroJSON example with Vector Layers
+The below example shows a TileJSON object for a MicroJSON tileset with multiple layers of detail. The tileset has a single vector layer, `image_layer` id of `vector_layers`, which contains a single vector layer describing images. The `fields` property of the this layer specifies the attributes of the layer, including the data types of the attributes. The `tiles` property specifies the URL pattern for accessing the vector tiles, which in this case is a 2D data set (no channels, time or z-axis) with zoom level. The `fillzoom` property specifies the zoom level from which to generate overzoomed tiles, which in this case starts at level 3, after the last specified layer.
+
+This file is located in the `examples/tiles` directory of the repository, and is named `tiled_example.json`. It has a corresponding MicroJSON file for each tile, located in the `examples/tiles/tiled_example` directory of the repository. The MicroJSON files are organized according to the tiling scheme, with the directory structure `zlvl/x/y.json` where `zlvl` is the zoom level, `x` is the x coordinate, and `y` is the y coordinate. The MicroJSON files contain the MicroJSON objects for the corresponding tiles, and are named according to the tiling scheme. For example, the MicroJSON object for the tile at zoom level 1, tile at (0,1) in the tiling scheme is located at `examples/tiles/tiled_example/1/0/1.json`. Examples for MicroJSON objects at zoom levels 0, 1, and 2 are provided below.
+
+```json
+{
+    "tilejson": "3.0.0",
+    "name": "2D Data Example",
+    "description": "A tileset showing 2D data with multiple layers of detail.",
+    "version": "1.0.0",
+    "attribution": "<a href='http://example.com'>Example</a>",
+    "tiles": [
+        "http://example.com/tiles/{zlvl}/{x}/{y}.json"
+    ],
+    "minzoom": 0,
+    "maxzoom": 10,
+    "bounds": [0, 0, 24000, 24000],
+    "center": [12000, 12000, 10],
+    "format": "json",
+    "vector_layers": [
+        {
+            "id": "image_layer",
+            "description": "Image layer",
+            "minzoom": 0,
+            "maxzoom": 10,
+            "fields": {
+                "string": {
+                    "Plate": "Plate ID",
+                    "Image": "Image URI"
+                },
+                "numerical": {
```

Comment:
Remove X and Y coordinates here if you are not showing in later in microjson example JSONs

Reply:
Removed